src/style.css

```
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+.form-field {
+  position: relative;
+  display: flex;
+  margin-top: 20px;
+  height: 70px;
+  width: 100%;
+}
+
+.small-form-field {
+  height: 42px;
+}
+
+.input-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.input-label::before {
+  position: absolute;
+  content: "";
+  background-image: url(./images/Search.svg);
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+  cursor: pointer;
+  z-index: 10;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.input-label-small::before {
+  height: 11px;
+  width: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.input {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  padding-top: 25px;
+  padding-left: 62px;
+  padding-bottom: 23px;
```

Do not hard code height, also you can use short hand
```suggestion
  padding: 26px 0 26px 62px;
```